source/parser.apib

```diff
@@ -32,7 +32,7 @@ application/vnd.refract.parse-result+json
 application/vnd.refract.parse-result+yaml
 ```
 
-General-purpose result of the parsing operation. The parse result is in form of the Refract data structure as defined in its [specification](https://github.com/refractproject/refract-spec). The parse result data comply with the [Parse Result Namespace](https://github.com/refractproject/refract-spec/blob/master/namespaces/parse-result.md).
+General-purpose result of the parsing operation. The parse result is in form of the Refract data structure as defined in its [specification](https://github.com/refractproject/refract-spec). The parse result data comply with the [Parse Result Namespace](https://github.com/refractproject/refract-spec/blob/master/namespaces/parse-result-namespace.md).
```

Comment:
Should probably link to API Elements instead?

http://api-elements.readthedocs.io/en/latest/